README.md

```diff
@@ -31,3 +31,25 @@ Upstream autoconf now
 For convenience when building packages that aren't yet updated, updated
 config.sub and config.guess files are installed at share/misc/config.\*
 in the install directory.
+
+## Notable Limitations
+
+This repository does not yet support C++ exceptions. C++ code is
+supported only with -fno-exceptions for now. Work on support for
+[exception handling] is underway at the language level.
```

Comment:
May also want to mention that setjmp/longjmp is not supported yet (and will also work with the exceptions feature).

Reply:
Good idea; I've now added a mention of this.